smt/surrogate_models/krg_based.py

```diff
@@ -1087,9 +1090,10 @@ def grad_minus_reduced_likelihood_function(log10t):
                             or self.noise0[i] > noise_bounds[1]
                         ):
                             self.noise0[i] = noise_bounds[0]
-                            print(
-                                "Warning: noise0 is out the feasible bounds. The lowest possible value is used instead."
-                            )
+                            default_warn = warnings.formatwarning
+                            warnings.formatwarning  = lambda msg, *args, **kwargs: str(msg)
+                            arnings.warn("Warning: noise0 is out the feasible bounds. The lowest possible value is used instead.\n")
```

Comment:
As @Paul-Saves mentioned, there is a typo here

Reply:
Thank you for spotting! I will update to just use warnings.warn() without the formatting